6_basic_simulation_analysis.ipynb

```diff
@@ -334,25 +334,6 @@
     "fig.show()"
    ]
   },
-  {
```

Comment:
@jdcourcol I'm removing this because the ffmpeg was not installed in Ebrains.
Do you prefer that I comment out in case they install it?

Reply:
I am fine to remove it